docs/dev/python-apis.md

```diff
@@ -1,3 +1,17 @@
 # How to Add Python APIs?
 
-![Under construction, coming soon!](../img/misc/under-construction.png)
\ No newline at end of file
+Python APIs in ShapeWorks are pybind wrappers of C++ functions. 
```

Comment:
Can you add details about the location of Python API files? Where can a new class/function be added?

Reply:
I second this request, otherwise I think this PR looks good

Reply:
Updated it with filename.